everything done on canned soup

morganbale opened the pull request:
@marcdotson I will start working on the modeling branch now and you can delete this one. I have pushed everything from my local computer.

marcdotson commented:
@morganbale there are a number of merge conflicts. It'll help following the revised repo structure:

- `Code` includes actual code, so the `.stan` files and the `.rds` compiled models.
- `Figures` is where all figures should go, and you'll want to name them (I'd recommend to match the corresponding chunks in your `.Rmd`) instead of relying on a whole bunch of unnamed files.
- `Writing` is where the actual writing goes, so basically any `.Rmd` files, whether they be the manuscript itself which knits to a PDF or one of these blog/vignette things that just knits as a GitHub document. It will of course draw from `Figures` and `Code` (well, probably just showing what's in `Code` since you won't want it to run when you knit everything again).

I may have borked my local repo trying to work through the merge conflicts. Can you make these changes to the `canned-soup-cleaning` branch, let me know, and I'll try again to merge?

marcdotson commented:
Include this at the beginning of your Rmd file after the YAML:

```
```{r opts, echo = FALSE}
knitr::opts_chunk$set(
  fig.path = "../Figures/folder_name/"
)
```
```

morganbale commented:
@marcdotson I named all the chunks and set the figures to save in the figures folder. I moved the model folder which has the .stan files to the code folder. I moved the .Rmd files to the main folder (not in any folder), because I only had a paper folder and not a writing folder. I thought you can move these into writing folder when you merge? Let me know if it doesnt work.
